main.go

```diff
@@ -2,53 +2,26 @@ package main
 
 import (
 	"fmt"
-	"os"
-
-	"github.com/clarifai/clarifai-go"
```

Comment:
@tomyang729  We never used it, so I am not sure why it was in the code, but initially we were gonna use it and then we figured that we can just use the API directly, because the library is poorly documented and it hasn't been updated for a while. I think it's even better, less dependancies, since we only hit one end point